ocpp/v201/enums.py

```diff
@@ -0,0 +1,1193 @@
+class Action:
```

Comment:
Can you subclass from `str` and `Enum`? [As is done for the OCPP 1.6 enums?](https://github.com/mobilityhouse/ocpp/blob/master/ocpp/v16/enums.py)